app/views/tags/edit.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#modal-tag-form').modal('show');
```

Comment:
Let's use a data-attribute for selecting the element rather than the ID. Please review the Frontend Development Guidelines on BC.